lib/elixir/test/elixir/process_test.exs

```diff
@@ -9,7 +9,9 @@ defmodule ProcessTest do
     assert Process.put(:foo, :bar) == nil
     assert Process.put(:foo, :baz) == :bar
 
-    assert Process.get_keys() == [:foo]
+    assert Enum.member?(Process.get_keys(), :foo)
+    refute Enum.member?(Process.get_keys(), :bar)
+    refute Enum.member?(Process.get_keys(), :baz)
```

Comment:
I had to change this to the tests, do you think that we should add an option like `ExUnit.Case generate_seed: false`, and then remove from this test this key?

Reply:
Let's update this test.

Reply:
To be clear, what you did is correct. :) We can ship it.